community/athens-guide/getting-started/installation-guide.md

```diff
@@ -1,22 +1,101 @@
+
+To find the location of this file for your operating system, click the **Choose Database** icon in the top-right corner of the Athens window.
+
+![Athens window with a red arrow pointing to the Choose Database icon](https://user-images.githubusercontent.com/15487/122690410-108d0b80-d1ef-11eb-8c4c-cd502b28e318.png)
+
+The file location appears in the **Open** tab that's displayed by default.
+
+![Choose Database modal with the current database file location marked with a red square](https://user-images.githubusercontent.com/15487/122767507-5cc86200-d268-11eb-93cb-858cc198320e.png)
+
+### Importing an existing database
+
+If you've used [Roam](https://roamresearch.com/) and want to bring your content into Athens, see documentation on [Athens import features](../feature-list/import.md).
+
+### Creating a new database
+
+To create a new database in a different location, click the **New** tab, enter a database name, and click the **Browse** button to select a directory to house the database files.
+
+![Choose Database modal with the New tab and its Database Name text box and Browse button marked by red arrows](https://user-images.githubusercontent.com/15487/122768180-f7c13c00-d268-11eb-959c-22bb2170c585.png)
+
+### Syncing your data between machines
 
 If you need to edit your knowledge graph from multiple machines or on different networks, you have two choices:
 
-* Use a file synchronization tool like Dropbox or OneDrive. While Athens warns you about updated blocks coming from these syncing tools, you should take care not to edit the same database from multiple machines at the same time.
-* Run the [Athens backend](https://github.com/athensresearch/athens-backend) on a system, like a virtual private server \(VPS\) or [Heroku deployment](https://github.com/athensresearch/athens-backend#heroku) that you can access from every machine or network.
+1. Use a file synchronization tool like [Dropbox](https://www.dropbox.com) or [OneDrive](https://www.microsoft.com/en-us/microsoft-365/onedrive/online-cloud-storage). While Athens warns you about updated blocks coming from these tools, you should take care not to edit the same database from multiple machines at the same time.
+2. Run the [Athens backend](https://github.com/athensresearch/athens-backend) on a system like a [virtual private server](https://en.wikipedia.org/wiki/Virtual_private_server) (VPS) or [Heroku deployment](https://github.com/athensresearch/athens-backend#heroku) that you can access from every machine or network. This backend is in the process of being refactored into the [main Athens git repository](https://github.com/athensresearch/athens) (see [this issue](https://github.com/athensresearch/athens/pull/1170/) for details), but until that effort is completed, the existing archived backend can be used.
```

Comment:
Let's remove the first sentence about the backend. We're actually not recommending that users install the backend any more, as the implementation isn't the best experience.